tutorials/custom_model.ipynb

```diff
@@ -0,0 +1,1037 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from matplotlib import pyplot as plt\n",
+    "from matplotlib.axes import Axes\n",
+    "from torch import Tensor\n",
+    "\n",
+    "\n",
+    "def plot_toy_data(x: Tensor, y: Tensor) -> Axes:\n",
+    "    ax = plt.figure().add_subplot(projection=\"3d\")\n",
```

Comment:
The 3d projection is cool!

Reply:
Yes, unfortunately I don't think the 3d projection is compatible with internal building. Will refactor this into a Y vs. Y_hat style plot and collapse the dimensions to 2 to make it compatible.